datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/mirrormaker/KafkaMirrorMakerConnectorTask.java

```diff
@@ -200,6 +200,10 @@ protected void consumerSubscribe() {
       // Consumer can be assigned with an empty set, but it cannot run poll against it
       // While it's allowed to assign an empty set here, the check on poll need to be performed
       _consumer.assign(_consumerAssignment);
+
+      // Invoke topic manager
```

Comment:
Might want to create a separate method for this and call that method? same code is being called from "onPartitionsAssigned" also

Reply:
consolidated them into handleTopicMangerPartitionAssignment